scripts/create_AR_DeployInst.py

```diff
@@ -114,6 +114,38 @@ def get_powers(path):
     return reactor_power
 
 
+def get_pris_powers(country, path, year):
+    '''
+    Create dictionary of the reactor units from a select country
+    in the PRIS database and
+    their corresponding rated power
+    output from the reactors_pris_XXXX.csv file for the corresponding year
+
+    Parameters:
+    -----------
+    country: str
+        name of country to get LWR data for
+    path: str
+        relative path to the pris csv file
+    year: int
+        year of data to pull from
+
+    Returns:
+    --------
+    pris_power: dict
+        dictionary of reactor names and rated powers, the keys are the reactor
+        names (strs), the values are the rated powers (ints). Any spaces
+        in the keys are replaced with underscores.
```

Comment:
```suggestion
    Parameters
    ----------
    country : str
        name of country to get LWR data for
    path : str
        relative path to the pris csv file
    year : int
        year of data to pull from

    Returns
    -------
    pris_power : dict
        dictionary of reactor names and rated powers, the keys are the reactor
        names (strs), the values are the rated powers (ints). Any spaces
        in the keys are replaced with underscores.
```

If you're gonna use numpy style, use numpy style

Reply:
I'm more worried about the docstrings being human readable than publishing the docstrings anywhere.